src/main/java/com/easypost/model/Shipment.java

```diff
@@ -8,287 +8,295 @@
-		Map<String, Object> wrappedParams = new HashMap<String, Object>();
-		wrappedParams.put("shipment", params);
-
-		return request(RequestMethod.POST, classURL(Shipment.class), wrappedParams, Shipment.class, apiKey);
-	}
-
-	// retrieve
-	public static Shipment retrieve(String id) throws EasyPostException {
-		return retrieve(id, null);
-	}
-	public static Shipment retrieve(String id, String apiKey) throws EasyPostException {
-		return request(RequestMethod.GET, instanceURL(Shipment.class, id), null, Shipment.class, apiKey);
-	}
-
-	// all
-	public static ShipmentCollection all(Map<String, Object> params) throws EasyPostException {
-		return all(params, null);
-	}
-	public static ShipmentCollection all(Map<String, Object> params, String apiKey) throws EasyPostException {
-		return request(RequestMethod.GET, classURL(Shipment.class), params, ShipmentCollection.class, apiKey);
-	}
-
-	// refresh
-	public Shipment refresh() throws EasyPostException {
-		return this.refresh(null, null);
-	}
-	public Shipment refresh(Map<String, Object> params) throws EasyPostException {
-		return this.refresh(params, null);
-	}
-	public Shipment refresh(String apiKey) throws EasyPostException {
-		return this.refresh((Map<String, Object>) null, apiKey);
-	}
-	public Shipment refresh(Map<String, Object> params, String apiKey) throws EasyPostException {
-		return request(
-			RequestMethod.GET,
-			String.format("%s", instanceURL(Shipment.class, this.getId())), params, Shipment.class, apiKey);
-	}
-
-	// get rates
-	public Shipment newRates() throws EasyPostException {
-		return this.newRates(null, null);
-	}
-	public Shipment newRates(Map<String, Object> params) throws EasyPostException {
-		return this.newRates(params, null);
-	}
-	public Shipment newRates(String apiKey) throws EasyPostException {
-		return this.newRates((Map<String, Object>) null, apiKey);
-	}
-	public Shipment newRates(Map<String, Object> params, String apiKey) throws EasyPostException {
-		Shipment response = request(
-			RequestMethod.POST,
-			String.format("%s/rerate", instanceURL(Shipment.class, this.getId())), params, Shipment.class, apiKey);
-
-		this.merge(this, response);
-		return this;
-	}
-
-	// get smartrates
+    public String id;
+    String mode;
+    String reference;
+    Boolean isReturn;
+    Address toAddress;
+    Address buyerAddress;
+    Address fromAddress;
+    Address returnAddress;
+    Parcel parcel;
+    CustomsInfo customsInfo;
+    Rate selectedRate;
+    List<Rate> rates;
+    PostageLabel postageLabel;
+    ScanForm scanForm;
+    String orderId;
+    List<Form> forms;
+    Tracker tracker;
+    String insurance;
+    String trackingCode;
+    String status;
+    String refundStatus;
+    String batchId;
+    String batchStatus;
+    String batchMessage;
+    String uspsZone;
+    Map<String, Object> options;
+    List<ShipmentMessage> messages;
+    List<TaxIdentifier> taxIdentifiers;
```

Comment:
# Reviewer Note

This is the main change, adding `taxIdentifiers` to the shipment object.